deployments/gpu-operator/templates/clusterpolicy.yaml

```diff
@@ -200,6 +200,9 @@ spec:
     {{- if .Values.driver.kernelModuleConfig }}
     kernelModuleConfig: {{ toYaml .Values.driver.kernelModuleConfig | nindent 6 }}
     {{- end }}
+    {{- if .Values.driver.secretEnv }}
+    secretEnv: {{ toYaml .Values.driver.secretEnv | nindent 6 }}
```

Comment:
Question -- is the `nindent 6` needed here if secretEnv is just a string?

Reply:
This is fixed